cmd/sovereignnode/incomingHeader/incomingEventsProcessor.go

```diff
@@ -4,23 +4,23 @@ import (
 	"encoding/hex"
 	"fmt"
 	"math/big"
-	"strings"
 
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/data"
 	"github.com/multiversx/mx-chain-core-go/data/smartContractResult"
 	"github.com/multiversx/mx-chain-core-go/hashing"
 	"github.com/multiversx/mx-chain-core-go/marshal"
+	"github.com/multiversx/mx-chain-go/sovereignnode/dataCodec"
 )
 
 const (
-	minTopicsInTransferEvent  = 4
+	minTopicsInTransferEvent  = 5
 	numTransferTopics         = 3
-	numExecutedBridgeOpTopics = 2
-	minNumEventDataTokens     = 4
+	numExecutedBridgeOpTopics = 3
 )
 
 const (
+	topicIDExecuteBridgeOps = "executeBridgeOps"
```

Comment:
is it ExecuteBridgeOps without a "d"? ExecuteD?

Reply:
These constants were modified...
We now have `event IDs` and `topic IDs`.

Event IDs are endpoints called in smart contracts.
Topic IDs is topics[0], aka identifier of the topics.

If this sounds bad, it's because how the explorer works, we can discuss this in the office :)